adaptive_navigation/example/lib/main.dart

```diff
@@ -9,24 +11,47 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Adaptive Navigation Scaffold',
-      home: AdaptiveNavigationScaffoldDemo(),
+      title: 'Adaptive Navigation Scaffold Demo',
+      home: DemoSelector(),
     );
   }
 }
 
-class AdaptiveNavigationScaffoldDemo extends StatelessWidget {
+class DemoSelector extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return AdaptiveNavigationScaffold(
-      currentIndex: 0,
-      destinations: [
-        AdaptiveScaffoldDestination(title: 'Home', icon: Icons.home),
-        AdaptiveScaffoldDestination(title: 'Settings', icon: Icons.settings),
-      ],
+    return Scaffold(
       body: Center(
-        child: Text(
-            'Resize the window to switch between the Navigation Rail and Bottom Navigation'),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            RaisedButton(
+              child: Text('Default Scaffold'),
+              onPressed: () {
+                Navigator.of(context).pushReplacement(
```

Comment:
Ah you're using pushReplacement here, so thats why you can't pop in the demos. Gotcha

Reply:
Yes, this is so that the new routes do not have default back buttons (see comment above for pushing '/' instead of pop)